README.md

```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| 13 | What is the appraisal cycle in the company?|
```

Comment:
This is not required!

Reply:
Yeah, this is way too overboard!

Reply:
It might look so but, this is probably the last question you ask to HR when you have multiple offers and are looking for better perks and future. Also asking this question gives you an idea about your future salary and increment percentage during appraisal. Also I do not think asking this question is overboad in any sense as a it is a fact that every company has an appraisal cycle, some annually and some bi annually or even quarterly and knowing these small details could affect your choice when choosing from multiple offers.

I have personally asked this question and it has helped me in choosing an offer